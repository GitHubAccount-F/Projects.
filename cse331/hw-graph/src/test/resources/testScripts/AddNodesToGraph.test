#Tests to see if nodes are added successfully

#Creates empty graphs
CreateGraph graph1
CreateGraph graph2
CreateGraph graph3

#Adds nodes to the graphs
AddNode graph2 one
AddNode graph3 one
AddNode graph3 two

#Lists the nodes in the graphs
ListNodes graph1
ListNodes graph2
ListNodes graph3