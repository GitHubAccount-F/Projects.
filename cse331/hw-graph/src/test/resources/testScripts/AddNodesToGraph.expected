#Tests to see if nodes are added successfully

#Creates empty graphs
created graph graph1
created graph graph2
created graph graph3

#Adds nodes to the graphs
added node one to graph2
added node one to graph3
added node two to graph3

#Lists the nodes in the graphs
graph1 contains:
graph2 contains: one
graph3 contains: one two